pkg/cli/admin/ocpcertificates/regeneratemco/command.go

```diff
@@ -0,0 +1,297 @@
+package regeneratemco
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"time"
+
+	configclient "github.com/openshift/client-go/config/clientset/versioned"
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/certrotation"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/spf13/cobra"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/informers"
+	"k8s.io/client-go/kubernetes"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	"k8s.io/kubectl/pkg/util/i18n"
+	"k8s.io/kubectl/pkg/util/templates"
+)
+
+const (
+	OneYear    = 365 * 24 * time.Hour
+	caExpiry   = 10 * OneYear
+	caRefresh  = 9 * OneYear
+	keyExpiry  = caExpiry
+	keyRefresh = caRefresh
+
+	mcoNamespace   = "openshift-machine-config-operator"
+	mapiNamespace  = "openshift-machine-api"
+	controllerName = "OcMachineConfigServerRotator"
+	mcsName        = "machine-config-server"
+
+	// mcsTlsSecretName is created by the installer and is not owned by default
+	mcsTlsSecretName = mcsName + "-tls"
+	// signerName is the name injected into the certificate as the signer
+	signerName = "oc-for-machine-config-server"
+	// openshiftOrg is the openshift organizational unit
+	openshiftOrg = "openshift"
+
+	// legacyRootCANamespace is the namespace of the legacy root CA
+	legacyRootCANamespace = "kube-system"
```

Comment:
we placed this in kube-system before?  How naughty of us.

Reply:
Unused. Go through all the consts and remove unused references.